driverdataaggregation/README.md

```diff
@@ -0,0 +1,407 @@
+      Default: 'aggregation-standard-scaling-py'
+      name of output dir
+  --launcher=LAUNCHER
+      Default: 'auto'
+      workload manager i.e. "slurm", "pbs"
+  --run_db_as_batch=RUN_DB_AS_BATCH
+      Default: True
+      run database as separate batch submission
+                                    each iteration
+  --db_node_feature=DB_NODE_FEATURE
+      Default: {}
+      dict of runsettings for db
+  --prod_node_feature=PROD_NODE_FEATURE
+      Default: {}
+      dict of runsettings for the producer
+  --cons_node_feature=CONS_NODE_FEATURE
+      Default: {}
+      dict of runsettings for the consumer
+  --db_hosts=DB_HOSTS
+      Default: []
+      optionally supply hosts to launch the database on
+  --db_nodes=DB_NODES
+      Default: [16]
+      number of compute hosts to use for the database
+  --db_cpus=DB_CPUS
+      Default: 36
+      number of cpus per compute host for the database
+  --db_port=DB_PORT
+      Default: 6780
+      port to use for the database
+  --net_ifname=NET_IFNAME
+      Default: 'ipogif0'
+      network interface to use i.e. "ib0" for infiniband or
+                            "ipogif0" aries networks
+  --clients_per_node=CLIENTS_PER_NODE
+      Default: [48]
+      client tasks per compute node for the aggregation
+                                    producer app
+  --client_nodes=CLIENT_NODES
+      Default: [128, 256, 512]
+      number of compute nodes to use for the aggregation
+                                producer app
+  --iterations=ITERATIONS
+      Default: 20
+      number of append/retrieve loops run by the applications
+  --tensor_bytes=TENSOR_BYTES
+      Default: [1024, 8192, 16384, 32769, 65538, 131076, 262152, 524304, 10...
+      list of tensor sizes in bytes
+  --tensors_per_dataset=TENSORS_PER_DATASET
+      Default: [1, 2, 4]
+      list of number of tensors per dataset
+  --client_threads=CLIENT_THREADS
+      Default: [1, 2, 4, 8, 16, 32]
+      list of the number of client threads used for data
+                                aggregation
+  --cpu_hyperthreads=CPU_HYPERTHREADS
+      Default: 2
+      the number of hyperthreads per cpu.  This is done
+                                    to request that the consumer application utilizes
+                                    all physical cores for each client thread
+    --languages=LANGUAGES
+        Default: ['cpp']
+        list of languages to use for the tester "cpp" and/or "fortran"
+    --wall_time=WALL_TIME
+        Default: '05:00:00'
+        allotted time for database launcher to run
+    --plot=PLOT
+        Default: 'database_nodes'
+        flag to plot against in process results
+```
+
+For example, the following command could be run to execute a battery of
+tests in the same allocation. The number of tests executed will be computed based
+on the number of permutations from the list inputs given.
+
+```bash
+python driver.py aggregation_scaling_python --exp_name='aggregation-scaling-py-batch' \
```

Comment:
I think you need to indent the code to make it show up as a bash script

Reply:
it shows up!